terraform/kubernetes.tf

```diff
@@ -9,22 +9,55 @@ resource "scaleway_k8s_cluster" "kube_cluster" {
       maintenance_window_start_hour = lookup(var.k8s_auto_upgrade_maintenance_window_start_hour, terraform.workspace, 3)
       maintenance_window_day = lookup(var.k8s_auto_upgrade_maintenance_window_day, terraform.workspace, "any")
    }
-
 }
 
-resource "scaleway_k8s_pool" "kube_nodepool" {
+# In this cluster, we create 2 nodepools :
+#
+# - The `default` nodepool, on which we'll deploy all pods that do not require
+#   autoscaling feature (e.g. the prometheus stack and basic jitsi components
+#   like prosody and jitsi-meet webserver). It is desirable to isolate them from
+#   nodepools with autoscaling enabled, as they can prevent the Cluster Autoscaler
+#   from removing a node if they run on them.
+#
+# - The `jibri` nodepool, dedicated to jibri instances, with autoscaling enabled.
+#
+# In the future, we'll propably add a nodepool dedicated to videobridges.
+#
+# This separation allows to choose instance types adapted to the needs of each
+# component (e.g. : Jibri consumes a lot of CPU/RAM, Videobridges consume a lot of
+# bandwdith).
+
+
+resource "scaleway_k8s_pool" "default" {
    autohealing         = lookup(var.k8s_nodepool_autohealing, terraform.workspace, true)
-   autoscaling         = lookup(var.k8s_nodepool_autoscale, terraform.workspace, false)
+   autoscaling         = false
    cluster_id          = scaleway_k8s_cluster.kube_cluster.id
    container_runtime   = lookup(var.k8s_nodepool_container_runtime, terraform.workspace, "containerd")
-   max_size            = lookup(var.k8s_nodepool_max_nodes, terraform.workspace, 1)
-   min_size            = lookup(var.k8s_nodepool_min_nodes, terraform.workspace, 1)
-   name                = "jitsi-${terraform.workspace}"
-   node_type           = lookup(var.k8s_nodepool_flavor, terraform.workspace, "GP1-XS")
-   size                = lookup(var.k8s_nodepool_size, terraform.workspace, 1)
+   name                = "default"
+   node_type           = lookup(var.k8s_default_nodepool_flavor, terraform.workspace, "GP1-XS")
+   size                = lookup(var.k8s_default_nodepool_size, terraform.workspace, 1)
    wait_for_pool_ready = true
 }
 
+
+resource "scaleway_k8s_pool" "jibri" {
+   autohealing         = lookup(var.k8s_nodepool_autohealing, terraform.workspace, true)
+   autoscaling         = lookup(var.k8s_jibri_nodepool_autoscale, terraform.workspace, true)
+   cluster_id          = scaleway_k8s_cluster.kube_cluster.id
+   container_runtime   = lookup(var.k8s_nodepool_container_runtime, terraform.workspace, "containerd")
+   max_size            = lookup(var.k8s_jibri_nodepool_max_nodes, terraform.workspace, 1)
+   min_size            = lookup(var.k8s_jibri_nodepool_min_nodes, terraform.workspace, 1)
+   name                = "jibri"
+   node_type           = lookup(var.k8s_jibri_nodepool_flavor, terraform.workspace, "GP1-XS")
```

Comment:
I think I remember you had problems with the smallest instance so maybe put a bigger one as default?

Reply:
On Scaleway? Are you sure?
This instance has 4 vCPUs and 16GB RAM.

Reply:
ok then